lib/result_builder.rb

```diff
@@ -1,33 +0,0 @@
-module StartingBlocks
-
-  class ResultBuilder
-
-    def build_from run_result
-      @run_result = run_result
-      {
-        color:     color,
-        text:      @run_result[:text],
-        exit_code: @run_result[:exit_code],
-        success:   @run_result[:success]
-      }
-    end
-
-    private
-
-    def color
-      return :red unless successful_exit_code?
-      return :red unless success?
-      :green
-    end
-
-    def successful_exit_code?
-      @run_result[:exit_code] == 0
-    end
-
-    def success?
-      @run_result[:success]
-    end
-
-  end
-end
-
```

Comment:
@rossfuhrman If you submitted this ```ResultBuilder``` as a PR to starting_blocks, I'd probably accept it.

I wrote the current one before considering the exit code, as I was ignorant of this basic fact. :smile:  My current one still inspects the counts of tests and all of that, but really... I just need to check the exit code.

Hint hint :eyes: wink wink :wink: